ReactWindows/ReactNative/ReactRootViewExtensions.cs

```diff
@@ -48,8 +48,7 @@ private static void OnAcceleratorKeyActivated(ReactNativeHost host, CoreDispatch
                     {
                         s_isControlKeyDown = e.EventType == CoreAcceleratorKeyEventType.KeyDown;
                     }
-                    else if ((s_isShiftKeyDown && e.VirtualKey == VirtualKey.F10) ||
```

Comment:
Shouldn't this be:
e.EventType == CoreAcceleratorKeyEventType.KeyDown &&
  ((s_isShiftKeyDown && e.VirtualKey == VirtualKey.F10) || 
  (! s_isControlKeyDown && e.VirtualKey == VirtualKey.Menu))

Reply:
Will change to
(e.EventType == CoreAcceleratorKeyEventType.KeyDown && s_isShiftKeyDown && e.VirtualKey == VirtualKey.F10)

When pressing ALT GR I get:
e.VirtualKey: Control and e.EventType: KeyDown
e.VirtualKey: Menu and e.EventType: KeyDown
e.VirtualKey: Control and e.EventType: SystemKeyUp
e.VirtualKey: Menu and e.EventType: KeyUp

So 
(CoreAcceleratorKeyEventType.KeyDown && ! s_isControlKeyDown && e.VirtualKey == VirtualKey.Menu) 
triggers the dev menu on ALT GR, and 
(CoreAcceleratorKeyEventType.KeyDown &&  s_isControlKeyDown && e.VirtualKey == VirtualKey.Menu) 
triggers the dev menu every other time I type ALT GR, I think because s_isControlKeyDown is not reset.

Isn't Shift + F10 enough to trigger the dev menu?

Reply:
Sorry, I confused with Application virtual key (the context menu button).
Yeah, there's no point to treat Alt key (with or without Ctrl) in any way different. Your fix looks good.

Reply:
Great! Thanks